functions/Invoke-DbaBalanceDataFiles.ps1

```diff
@@ -84,113 +92,109 @@ function Invoke-DbaBalanceDataFiles {
 	By supplying this parameter you give permission to do the rebuilds offline if the edition does not support it.
 
 #>
-	[CmdletBinding(DefaultParameterSetName = "Default", SupportsShouldProcess = $true)]
+	[CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = 'High', DefaultParameterSetName = "Default")]
 	param (
-		[parameter(ParameterSetName = "Pipe", Mandatory = $true)]
+		[parameter(Mandatory, ParameterSetName = "instance")]
+		[Alias("ServerInstance", "SqlServer")]
 		[DbaInstanceParameter[]]$SqlInstance,
```

Comment:
This should just be `[DbaInstance]`. I've been removing the `DbaInstanceParameter` as I touch commands so we can get rid of it :tada: